web/src/SidebarIcon.tsx

```diff
@@ -91,23 +90,16 @@ export default class SidebarIcon extends PureComponent<SidebarIconProps> {
       icon = <CheckmarkSmallSvg role="presentation" />
     }
 
-    if (!this.props.tooltipText.length) {
-      return (
-        <SidebarIconRoot
-          className={`${ClassNameFromResourceStatus(this.props.status)}`}
-        >
-          {icon}
-        </SidebarIconRoot>
-      )
-    }
-    return (
-      <Tooltip title={this.props.tooltipText}>
-        <SidebarIconRoot
-          className={`${ClassNameFromResourceStatus(this.props.status)}`}
-        >
-          {icon}
-        </SidebarIconRoot>
-      </Tooltip>
+    let content = (
+      <SidebarIconRoot
+        className={`${ClassNameFromResourceStatus(this.props.status)}`}
+      >
+        {icon}
+      </SidebarIconRoot>
     )
+    if (!this.props.tooltipText) {
+      return content
+    }
+    return <Tooltip title={this.props.tooltipText}>{content}</Tooltip>
```

Comment:
I'm pretty sure MUI's Tooltip bows out and just returns its children when `title` is empty, which would save us from this whole dance. I don't know if we failed to match that behavior in Tilt for some reason or were just unaware when we wrote this code.

This PR is an improvement either way, though.

Reply:
oops good point!